releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -6,6 +6,13 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 1.1.3/1.2.3 - April 06, 2020
```

Comment:
Are there any github issues related to any of these fixes? If so, it would be good to reference them in the release notes as they provide extra information if anyone is interested.